features/yield/FarmingCalculator.tsx

```diff
@@ -240,17 +274,17 @@ const FarmingCalculator = () => {
           </a>
           .
         </Typography>
-        {hoursRemainingToFarmingRoll > 0 && (
+        {timeUntilRoll && timeUntilRoll > 0 && (
           <>
             <br></br>
             <br></br>
             <Typography>
-              <strong>Update (08/24 @ 23:00 UTC):</strong> Before August 28th,
-              23:00 UTC, LP contributions to either the yUSD-SEP20 or the
-              yUSD-OCT20 are considered equally. What this means is that UMA
-              rewards are granted pro-rata as: (your USD contribution) / (total
-              SEP20 liquidity + total OCT20 liquidity). After August 28, 23:00
-              UTC, only LP contributions in the OCT20 pool will count towards
+              <strong>Update (09/23 @ 23:00 UTC):</strong> Before September
+              23rd, 23:00 UTC, LP contributions to either the OCT20 or the NOV20
+              are considered equally. What this means is that UMA rewards are
+              granted pro-rata as: (your USD contribution) / (total OCT20
+              liquidity + total NOV20 liquidity). After September 27, 23:00 UTC,
+              only LP contributions in the NOV20 pool will count towards
```

Comment:
One thing that's a little confusing to me is that we're encoding all these times in code everywhere else, but we're hardcoding them in this text. Is there a quick way we can describe the general process of the roll and then add a list or table of applicable tokens with their roll status. This may be annoying, though, so I wouldn't worry about it unless you can think of a quick way to encode this.

Reply:
Another option could be to store these strings in the `ROLL_REWARDS_SCHEDULE` object and then pull them out in the sentences.

Reply:
Will remove the hard codes

Reply:
I now read the roll date dynamically